src/formatDate.js

```diff
@@ -8,7 +8,61 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.find(
+    (item) => item === '/' || item === '-' || item === '.',
```

Comment:
The logic for finding the `fromSeparator` is incorrect. The `find` method is being used with a function that checks if an item is a separator, but `find` should be used to find a separator in the `date` string, not in the `fromFormat` array. Consider revising this logic to correctly identify the separator used in the `date` string.